src/graphql/typeDefs/dataTypes/eventInput.graphql

```diff
@@ -46,6 +46,9 @@ input EventCreateInput {
 
   "The community this Event is part of"
   community: String
+
+  "Represents the overall color theme for this event"
```

Comment:
I think we're missing some mutations that allow us to create and update these types.

Reply:
@csell5  As in a mutation resolver? Not needed they are mutated along with the event resolver.

Reply:
e.g.

```graphql
mutation createEvent ($event: EventCreateInput!) {
   events {
    create(event: $event) {
      id
      name
      slug
    }
  }
}
```

```json
{
	"event": {
		"name": "THAT Conference 2022",
		"slug": "wi-test/2022",
		"slogan": "Summer Camp For Geeks",
		"description": "Over four days, folks of diverse technology...",
		"startDate": "2022-07-25T12:09:39.343Z",
		"endDate": "2022-07-28T12:09:39.343Z",
		"year": 2022,
		"email": "Hello@THATConference.com",
		"website": "https://www.thatconference.com/",
		"theme": {
			"heroSlug": "images/hero.jpg",
			"primary": "#111111",
			"secondary": "#222222",
			"tertiary": "#333333",
			"highlight": "#444444"
		},
		"community": "other"
	}
}
```